system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -592,14 +592,14 @@ void CLdapSecManager::init(const char *serviceName, IPropertyTree* cfg)
 
     m_ldap_client.setown(ldap_client);
     m_pp.setown(pp);
-    int cachetimeout = cfg->getPropInt("@cacheTimeout", 5);
+    int cacheTimeoutMinutes = cfg->getPropInt("@cacheTimeout", 60);//config value is in minutes
```

Comment:
Is it possible to #define the 60 in one location and use it also in caching.hpp?